projects/dev-schematics/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@spartacus/dev-schematics",
+  "version": "0.0.1",
+  "description": "Spartacus dev-schematics",
+  "keywords": [
+    "spartacus",
+    "dev-schematics"
+  ],
+  "scripts": {
+    "clean": "rm -rf src/**/*.js src/**/*.js.map src/**/*.d.ts",
+    "build": "yarn clean && tsc -p tsconfig.json",
+    "test": "tsc -p tsconfig.json && jasmine \"src/**/*_spec.js\""
```

Comment:
Let's also update this one to: `"test": "yarn build && jasmine \"src/**/*_spec.js\""`